__tests__/components/InputBox.test.tsx

```diff
@@ -0,0 +1,40 @@
+import InputBox from '@/components/InputBox';
+import { fireEvent, render } from '@testing-library/react';
+
+describe('InputBox', () => {
+    it('should render', () => {
+        const { container } = render(
```

Comment:
should render what??

Reply:
Inputbox, mentioned in the describe

Reply:
sir, dont use shortcuts here... `should render input box looks much better then simply should render